modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -1080,10 +1080,22 @@ class Editor implements EditorObservable {
     VisualAids.addVisual(this, elm);
   }
 
+  /**
+   * Changes the editable state of the editor root element.
+   *
+   * @method setEditableRoot
+   * @param {Boolean} state State to set true for editable and false for non-editable.
+   */
   public setEditableRoot(state: boolean): void {
     EditableRoot.setEditableRoot(this, state);
   }
 
+  /**
+   * Returns the current editable state of the editor root element.
+   *
+   * @method hasEditableRoot
+   * @return {Boolean} True if the root element is editable, false if it's not editable.
```

Comment:
```suggestion
   * Returns the current editable state of the editor’s root element.
   *
   * @method hasEditableRoot
   * @return {Boolean} True if the root element is editable, false if it is not editable.
```
1. Inflect for the possessive case with the Saxon genitive.
2. expand _it’s_ to _it is_ for formal clarity.

Reply:
Makes sense. Changed this.